dll/cpl/desk/lang/fr-FR.rc

```diff
@@ -80,23 +80,23 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL "", IDC_APPEARANCE_PREVIEW, "PreviewWndClass", WS_VISIBLE | WS_BORDER, 7, 7, 232, 120
     LTEXT "Si vous sélectionnez un paramètre de boutons et de fenêtres autre que ReactOS classique, celui-ci va remplacer les paramètres suivants exception faite de certains programmes plus anciens.", IDC_ADVAPPEARANCE_DISCLAIMER, 7, 133, 232, 30, BS_MULTILINE
-    LTEXT "Élément :", 8, 5, 160, 50, 9
+    LTEXT "Élém&ent :", 8, 5, 160, 50, 9
     COMBOBOX IDC_ADVAPPEARANCE_ELEMENT, 5, 170, 130, 90, CBS_DROPDOWNLIST |
              CBS_HASSTRINGS | CBS_SORT | WS_CHILD | WS_VSCROLL | WS_TABSTOP
     LTEXT "&Taille :", IDC_ADVAPPEARANCE_SIZE_T, 142, 160, 39, 9
     EDITTEXT IDC_ADVAPPEARANCE_SIZE_E, 142, 170, 38, 13, ES_RIGHT | WS_GROUP
     CONTROL "", IDC_ADVAPPEARANCE_SIZE_UD, UPDOWN_CLASS, UDS_AUTOBUDDY | WS_BORDER | WS_GROUP |
             UDS_ALIGNRIGHT | UDS_ARROWKEYS | UDS_SETBUDDYINT | WS_CHILD | WS_VISIBLE, 172, 170, 10, 13
-    LTEXT "Couleur 1 :", IDC_ADVAPPEARANCE_COLOR1_T, 180, 160, 34, 9
+    LTEXT "Coule&ur 1 :", IDC_ADVAPPEARANCE_COLOR1_T, 180, 160, 34, 9//FIXME: control is too small to hold all the text. For the backport I simply stripped all " :" in this dlg. Imho that would be a good solution also for master. Some other translations do that as well.
     CHECKBOX "-", IDC_ADVAPPEARANCE_COLOR1_B, 184, 170, 28, 13, BS_PUSHLIKE | BS_BITMAP
-    LTEXT "Couleur 2 :", IDC_ADVAPPEARANCE_COLOR2_T, 215, 160, 34, 9
+    LTEXT "C&ouleur 2 :", IDC_ADVAPPEARANCE_COLOR2_T, 215, 160, 34, 9//FIXME: control is too small to hold all the text. For the backport I simply stripped all " :" in this dlg. Imho that would be a good solution also for master. Some other translations do that as well.
     CHECKBOX "-", IDC_ADVAPPEARANCE_COLOR2_B, 218, 170, 28, 13, BS_PUSHLIKE | BS_BITMAP
-    LTEXT "Police :", IDC_ADVAPPEARANCE_FONT_T, 5, 190, 50, 9
+    LTEXT "&Police :", IDC_ADVAPPEARANCE_FONT_T, 5, 190, 50, 9
     COMBOBOX IDC_ADVAPPEARANCE_FONT_C, 5, 200, 130, 90, CBS_DROPDOWN |
              CBS_HASSTRINGS | CBS_SORT | WS_VSCROLL | WS_TABSTOP
-    LTEXT "Taille :", IDC_ADVAPPEARANCE_FONTSIZE_T, 142, 190, 39, 9
+    LTEXT "T&aille :", IDC_ADVAPPEARANCE_FONTSIZE_T, 142, 190, 39, 9
     COMBOBOX IDC_ADVAPPEARANCE_FONTSIZE_E, 142, 200, 38, 200, CBS_DROPDOWN | WS_VSCROLL
-    LTEXT "Couleur :", IDC_ADVAPPEARANCE_FONTCOLOR_T, 184, 190, 28, 9
+    LTEXT "&Couleur :", IDC_ADVAPPEARANCE_FONTCOLOR_T, 184, 190, 28, 9//FIXME: control is too small to hold all the text. For the backport I simply stripped all " :" in this dlg. Imho that would be a good solution also for master. Some other translations do that as well.
```

Comment:
```suggestion
    LTEXT "&Couleur :", IDC_ADVAPPEARANCE_FONTCOLOR_T, 184, 190, 30, 9
```

Reply:
no, please do that with your own PR later.